src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+        area();
```

Comment:
why do we need this?

Reply:
To creat an object. And color it`s a status of object so I made a constructor in abstract maethod Figure to avoid every time writing this.color = color I use alredy created  in parent class